languages/common-lisp/exercises/concept/sameness/.docs/hints.md

```diff
@@ -0,0 +1,23 @@
+## 1. The maze of object equality
+
+This maze needs the most restrictive equality predicate available.
```

Comment:
The individual hints should be prefixed with `- `:

```suggestion
- This maze needs the most restrictive equality predicate available.
````
 
This also applies to the other hints in this file.

Reply:
We often link to external resources in the hints file, to directly point students towards the right method/function to use (but without spoiling the solution in this file). Is there a document you could link to that contains information on the equality predicate that should be used?

This comment also applies to the other hints in this file.

Reply:
I think any link provided would be to direct pages on the equality predicates - and thus tell the student want to do. But I will open an issue and do some digging for a better less specific resource. There is one I link to in the `after.md` which might be good.